gamatrix-gog.py

```diff
@@ -337,7 +334,6 @@ def set_multiplayer_status(game_list, cache):
       - IGDB game modes includes a multiplayer mode
     """
     for k in game_list:
-        igdb_key = game_list[k]["igdb_key"]
```

Comment:
Looks like I hosed you with my merge last night. You'll need to merge `master` in.